week3/homework/nodejs-exercises/exercise-3/server.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+
+const data = {
+  name: 'Name Surname',
+  numberOfPeople: 12,
+};
+
+axios({
+  method: 'post',
+  url: 'https://reservation100-sandbox.mxapps.io/api/reservations',
+  data: data,

```

Comment:
js is smart enough to know the key and the value variable name are the same so in this case you can do:
```suggestion
  data,
```